collectors/python.d.plugin/web_log/README.md

```diff
@@ -1,4 +1,6 @@
-# Web log monitoring with Netdata
+# Apache/NGINX/Squid monitoring with Netdata
```

Comment:
I do think we should mention that this is _log_ monitoring, and thus different from the Apache/Nginx modules.

```suggestion
# Apache/NGINX/Squid log monitoring with Netdata
```